README.md

```diff
@@ -82,6 +82,26 @@ And the following **modules** are exposed:
 
 Note that `commonTools.letsEncrypt` is not available anymore as the next controller won't support it (use `@iobroker/webserver` instead).
 
+## i18n
+Developer can use internationalisation in backend.
+
+For that call
+```javascript
+const i18n = require('@iobroker/adapter-core').i18n;
+
+// later in "ready" method
+i18n.init(__dirname, adapter);
+// If you use class syntax, you can use `this` instead of `adapter`
+i18n.init(__dirname, this);
+```
+
+and then in code
+```javascript
+  console.log(i18n.t('text to translate %s', 'argument1'));
```

Comment:
t? in implementation it is "translate"? Same for next line?

Reply:
Also dodess th build match to this .ts code? Yes in build it is "t" and "tt" ... but not in the TS file?!

Reply:
Changed